conda-recipe/meta.yaml

```diff
@@ -17,12 +17,14 @@ requirements:
     - numpy
     - six
     - matplotlib
+    - enaml
```

Comment:
@sameera2004 Correct on the conda side, I don't know the if enaml is needed to build bubble gum though.  Things in bulid are not automatically included in the run deps.  ex you only need a complier at build time, but not at run time.